File: dsa/array_string_obj/maxMinElementsInArray.js
//Find max and min in array(Math.max(...arr), Math.min(...arr))
function getMinMax(arr) {
  let min = Infinity,
    max = -Infinity;
    
  if (!Array.isArray(arr) || arr.some(x => !Number.isFinite(x))) return false;
  if (arr.length === 0) return null;

  for (let i = 0; i < arr.length; i++) {
    if (arr[i] < min) {
      min = arr[i];
    }
    if (arr[i] > max) {
      max = arr[i];
    }
  }
  return [min, max];
}
console.log(getMinMax([5,2,8,1]))

File: dsa/array_string_obj/maxMinElementsInArray.py
def get_min_max(arr):
    min_val = float('inf')
    max_val = float('-inf')
    for num in arr:
        if num < min_val:
            min_val = num
        if num > max_val:
            max_val = num
    return min_val, max_val

print(get_min_max([5, 2, 8, 1]))
